docs/docusaurus.config.js

```diff
@@ -121,7 +121,7 @@ const config = {
 						position: 'left',
 					},
 					{
-						to: 'api', // 'api' is the 'out' directory
+						to: 'docs/api', // 'api' is the 'out' directory
```

Comment:
![image](https://user-images.githubusercontent.com/86780488/182593575-090b0967-0509-44cd-b41d-7c09435df87e.png)
API link in header also needs update